UnitTests/TimeMachineUnitTests.cs

```diff
@@ -1,4 +1,5 @@
-﻿using NUnit.Framework;
+﻿using System;
+using NUnit.Framework;
 using UMLEditor.Classes;
 using UMLEditor.Exceptions;
 
```

Comment:
Making d1, d2, and d3 not nullable in the unit tests will help with some of the warnings. The rest will need to be addressed in a later refactor pass.